### ISSUE TITLE
v0.108.0-b.50 memory leak

### ISSUE THREAD

**Issue by timkgh** [AdguardTeam/AdGuardHome#6476]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, MIPS64

### Installation

GitHub releases or script from README

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

v0.108.0-b.50

### Action

Just let it run normally

### Expected result

Do not leak memory

### Actual result

Every few days I see these messages on my router:
```
[Tue Nov 28 11:19:05 2023] Out of memory: Kill process 30568 (AdGuardHome) score 735 or sacrifice child
[Tue Nov 28 11:19:05 2023] Killed process 30568 (AdGuardHome) total-vm:1429384kB, anon-rss:730464kB, file-rss:0kB, shmem-rss:892kB
[Tue Nov 28 11:19:06 2023] oom_reaper: reaped process 30568 (AdGuardHome), now anon-rss:0kB, file-rss:0kB, shmem-rss:892kB
```

The AdGuardHome process was using over 900MB of memory when all my router has is 1GB. After a restart it uses 60MB and updating the filters does not seem to increase the memory use. I suspect there's something else that's growing unbounded, caches, etc.

Note that this may have started with this version, I haven't seen a memory leak in a while, though I did experience them with AGH before.

Possibly related https://github.com/AdguardTeam/AdGuardHome/issues/3913

**Comment by ishanjain28** [comment]
One of my adguard home instances is also on, `v0.108.0-b.50` and it has the same problem.
![image](https://github.com/AdguardTeam/AdGuardHome/assets/7921368/1d9ccb09-9b39-4acd-85c0-2185f2a0ea44)


Instance logs,

```
Nov 27 04:20:24 adguardhome systemd[1]: Started AdGuardHome.service - AdGuard Home: Network-level blocker.
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.893460 [info] AdGuard Home, version v0.108.0-b.50
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.893476 [info] AdGuard Home is running as a service
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.895507 [info] tls: using default ciphers
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.896242 [info] safesearch default: disabled
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.899457 [info] Initializing auth module: /root/data/sessions.db
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.899618 [info] auth: initialized.  users:1  sessions:9
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.900113 [info] tls: number of certs: 2
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.900127 [info] tls: got an intermediate cert
Nov 27 04:20:24 adguardhome AdGuardHome[262]: 2023/11/27 04:20:24.930149 [info] web: initializing
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.091874 [info] dnsproxy: cache: enabled, size 4096 b
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.091890 [info] dnsproxy: max goroutines is set to 50
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092134 [info] AdGuard Home is available at the following addresses:
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092158 [info] clients: processing addresses
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092189 [info] go to http://127.0.0.1:80
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092193 [info] go to http://[::1]:80
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092195 [info] go to http://10.0.99.14:80
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092197 [info] go to http://[2a0a:6040:4004:99::a1]:80
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092200 [info] go to http://[fe80::2805:eeff:fe40:4299%eth0]:80
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.092549 [info] go to https://dns.ishan.pw:443
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.093374 failed to sufficiently increase send buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details.
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378537 [info] dnsproxy: starting dns proxy server
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378573 [info] Cache TTL override is enabled. Min=1800, Max=3600
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378578 [info] Ratelimit is enabled and set to 400 rps, IPv4 subnet mask len 24, IPv6 subnet mask len 56
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378581 [info] The server is configured to refuse ANY requests
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378584 [info] dnsproxy: cache: enabled, size 1024000 b
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378643 [info] dnsproxy: max goroutines is set to 50
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378657 [info] dnsproxy: creating udp server socket 0.0.0.0:53
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378700 [info] dnsproxy: listening to udp://[::]:53
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378705 [info] dnsproxy: creating tcp server socket 0.0.0.0:53
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378721 [info] dnsproxy: listening to tcp://[::]:53
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378723 [info] dnsproxy: creating tls server socket 0.0.0.0:853
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378730 [info] dnsproxy: listening to tls://[::]:853
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.378734 [info] Creating a QUIC listener
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.379004 [info] Listening to quic://[::]:853
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.379230 [info] dnsproxy: entering tcp listener loop on [::]:53
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.379259 [info] dnsproxy: entering udp listener loop on [::]:53
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.379289 [info] dnsproxy: entering tls listener loop on [::]:853
Nov 27 04:20:25 adguardhome AdGuardHome[262]: 2023/11/27 04:20:25.379296 [info] Entering the DNS-over-QUIC listener loop on [::]:853
Nov 27 04:20:27 adguardhome AdGuardHome[262]: 2023/11/27 04:20:27.825935 [info] home: language is set to en
Nov 27 04:20:27 adguardhome AdGuardHome[262]: 2023/11/27 04:20:27.826196 [info] home: theme is set to light
Nov 27 04:20:37 adguardhome AdGuardHome[262]: 2023/11/27 04:20:37.775850 [error] POST 10.0.99.14 /control/version.json: getting version info from https://static.adtidy.org/adguardhome/beta/version.json: updater: HTTP GET https://static.adtidy.org/adguardhome/beta/version.json: Get "https://static.adtidy.org/adguardhome/beta/version.json": resolving "static.adtidy.org": getting conn to tls://dns10.quad9.net:853: creating dial handler: dialing "dns10.quad9.net:853": resolving hostname: lookup dns10.quad9.net: i/o timeout
                                              getting conn to tls://one.one.one.one:853: creating dial handler: dialing "one.one.one.one:853": resolving hostname: lookup one.one.one.one: i/o timeout
                                              getting conn to tls://dns.google:853: creating dial handler: dialing "dns.google:853": resolving hostname: lookup dns.google: i/o timeout
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.402616 [info] dnsforward: starting reconfiguring server
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.402874 [info] dnsproxy: stopping dns proxy server
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.402992 [info] dnsproxy: stopped dns proxy server
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.503748 [info] dnsproxy: cache: enabled, size 4096 b
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.503765 [info] dnsproxy: max goroutines is set to 50
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505080 [info] dnsproxy: starting dns proxy server
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505094 [info] Cache TTL override is enabled. Min=1800, Max=3600
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505097 [info] Ratelimit is enabled and set to 400 rps, IPv4 subnet mask len 24, IPv6 subnet mask len 56
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505100 [info] The server is configured to refuse ANY requests
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505102 [info] dnsproxy: cache: enabled, size 1024000 b
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505105 [info] dnsproxy: max goroutines is set to 50
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505111 [info] dnsproxy: creating udp server socket 0.0.0.0:53
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505171 [info] clients: processing addresses
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505375 [info] dnsproxy: listening to udp://[::]:53
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505379 [info] dnsproxy: creating tcp server socket 0.0.0.0:53
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505392 [info] dnsproxy: listening to tcp://[::]:53
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505395 [info] dnsproxy: creating tls server socket 0.0.0.0:853
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505404 [info] dnsproxy: listening to tls://[::]:853
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505406 [info] Creating a QUIC listener
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505486 [info] Listening to quic://[::]:853
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505491 [info] dnsforward: finished reconfiguring server
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505631 [info] dnsproxy: entering udp listener loop on [::]:53
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505869 [info] dnsproxy: entering tcp listener loop on [::]:53
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505880 [info] dnsproxy: entering tls listener loop on [::]:853
Nov 27 04:21:35 adguardhome AdGuardHome[262]: 2023/11/27 04:21:35.505885 [info] Entering the DNS-over-QUIC listener loop on [::]:853
Nov 27 04:22:45 adguardhome AdGuardHome[262]: 2023/11/27 04:22:45.761508 [error] handling tcp: reading msg: reading len: read tcp 10.0.99.14:53->10.0.10.33:50344: read: connection reset by peer
Nov 27 04:23:11 adguardhome AdGuardHome[262]: 2023/11/27 04:23:11.344333 [info] home: language is set to en
Nov 27 04:23:11 adguardhome AdGuardHome[262]: 2023/11/27 04:23:11.344689 [info] home: theme is set to light
Nov 27 04:30:36 adguardhome AdGuardHome[262]: 2023/11/27 04:30:36.303058 [info] home: language is set to en
Nov 27 04:30:36 adguardhome AdGuardHome[262]: 2023/11/27 04:30:36.303363 [info] home: theme is set to light
Nov 27 04:42:59 adguardhome AdGuardHome[262]: 2023/11/27 04:42:59.354797 [info] home: language is set to en
Nov 27 04:42:59 adguardhome AdGuardHome[262]: 2023/11/27 04:42:59.355042 [info] home: theme is set to light
Nov 27 04:54:17 adguardhome AdGuardHome[262]: 2023/11/27 04:54:17.259262 [info] home: language is set to en
Nov 27 04:54:17 adguardhome AdGuardHome[262]: 2023/11/27 04:54:17.259281 [info] home: theme is set to light
Nov 27 06:20:39 adguardhome AdGuardHome[262]: 2023/11/27 06:20:39.300343 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 27 06:20:39 adguardhome AdGuardHome[262]: 2023/11/27 06:20:39.303662 [info] filtering: updated filter 1563800924: 1146154 bytes, 58421 rules
Nov 27 06:20:45 adguardhome AdGuardHome[262]: 2023/11/27 06:20:45.383761 [info] filtering: updated filter 1563800924; rule count: 58421 (was 58420)
Nov 27 09:21:07 adguardhome AdGuardHome[262]: 2023/11/27 09:21:07.870321 [info] filtering: saving contents of filter 1606027806 into "/root/data/filters/1606027806.txt"
Nov 27 09:21:07 adguardhome AdGuardHome[262]: 2023/11/27 09:21:07.873320 [info] filtering: updated filter 1606027806: 527022 bytes, 19958 rules
Nov 27 09:21:08 adguardhome AdGuardHome[262]: 2023/11/27 09:21:08.508278 [info] filtering: updated filter 1606027806; rule count: 19958 (was 19952)
Nov 27 12:21:23 adguardhome AdGuardHome[262]: 2023/11/27 12:21:23.894246 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 27 12:21:23 adguardhome AdGuardHome[262]: 2023/11/27 12:21:23.899183 [info] filtering: updated filter 1563800924: 1147021 bytes, 58464 rules
Nov 27 12:21:29 adguardhome AdGuardHome[262]: 2023/11/27 12:21:29.547065 [info] filtering: updated filter 1563800924; rule count: 58464 (was 58421)
Nov 27 14:21:46 adguardhome AdGuardHome[262]: 2023/11/27 14:21:46.064424 [info] filtering: saving contents of filter 1606027803 into "/root/data/filters/1606027803.txt"
Nov 27 14:21:46 adguardhome AdGuardHome[262]: 2023/11/27 14:21:46.067123 [info] filtering: updated filter 1606027803: 959875 bytes, 57855 rules
Nov 27 14:21:47 adguardhome AdGuardHome[262]: 2023/11/27 14:21:47.487026 [info] filtering: updated filter 1606027803; rule count: 57855 (was 57561)
Nov 27 16:58:29 adguardhome AdGuardHome[262]: 2023/11/27 16:58:29.008374 [info] home: language is set to en
Nov 27 16:58:29 adguardhome AdGuardHome[262]: 2023/11/27 16:58:29.008602 [info] home: theme is set to light
Nov 27 17:45:10 adguardhome AdGuardHome[262]: 2023/11/27 17:45:10.634269 [info] home: language is set to en
Nov 27 17:45:10 adguardhome AdGuardHome[262]: 2023/11/27 17:45:10.634509 [info] home: theme is set to light
Nov 27 18:19:00 adguardhome AdGuardHome[262]: 2023/11/27 18:19:00.943041 [info] home: language is set to en
Nov 27 18:19:00 adguardhome AdGuardHome[262]: 2023/11/27 18:19:00.943403 [info] home: theme is set to light
Nov 27 18:22:29 adguardhome AdGuardHome[262]: 2023/11/27 18:22:29.567712 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 27 18:22:29 adguardhome AdGuardHome[262]: 2023/11/27 18:22:29.571792 [info] filtering: updated filter 1563800924: 1149244 bytes, 58554 rules
Nov 27 18:22:35 adguardhome AdGuardHome[262]: 2023/11/27 18:22:35.744457 [info] filtering: updated filter 1563800924; rule count: 58554 (was 58464)
Nov 27 20:17:16 adguardhome AdGuardHome[262]: 2023/11/27 20:17:16.891112 [info] home: language is set to en
Nov 27 20:17:16 adguardhome AdGuardHome[262]: 2023/11/27 20:17:16.891525 [info] home: theme is set to light
Nov 28 00:23:17 adguardhome AdGuardHome[262]: 2023/11/28 00:23:17.041867 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 28 00:23:17 adguardhome AdGuardHome[262]: 2023/11/28 00:23:17.045856 [info] filtering: updated filter 1563800924: 1138022 bytes, 58023 rules
Nov 28 00:23:23 adguardhome AdGuardHome[262]: 2023/11/28 00:23:23.307461 [info] filtering: updated filter 1563800924; rule count: 58023 (was 58554)
Nov 28 04:23:52 adguardhome AdGuardHome[262]: 2023/11/28 04:23:52.620906 [info] filtering: saving contents of filter 1606027806 into "/root/data/filters/1606027806.txt"
Nov 28 04:23:52 adguardhome AdGuardHome[262]: 2023/11/28 04:23:52.622472 [info] filtering: updated filter 1606027806: 527086 bytes, 19960 rules
Nov 28 04:23:53 adguardhome AdGuardHome[262]: 2023/11/28 04:23:53.286193 [info] filtering: updated filter 1606027806; rule count: 19960 (was 19958)
Nov 28 06:24:02 adguardhome AdGuardHome[262]: 2023/11/28 06:24:02.398132 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 28 06:24:02 adguardhome AdGuardHome[262]: 2023/11/28 06:24:02.400898 [info] filtering: updated filter 1563800924: 1139112 bytes, 58079 rules
Nov 28 06:24:09 adguardhome AdGuardHome[262]: 2023/11/28 06:24:09.056926 [info] filtering: updated filter 1563800924; rule count: 58079 (was 58023)
Nov 28 06:33:02 adguardhome AdGuardHome[262]: 2023/11/28 06:33:02.586078 [info] home: language is set to en
Nov 28 06:33:02 adguardhome AdGuardHome[262]: 2023/11/28 06:33:02.586618 [info] home: theme is set to light
Nov 28 07:09:02 adguardhome AdGuardHome[262]: 2023/11/28 07:09:02.650636 [info] home: language is set to en
Nov 28 07:09:02 adguardhome AdGuardHome[262]: 2023/11/28 07:09:02.651547 [info] home: theme is set to light
Nov 28 11:41:36 adguardhome AdGuardHome[262]: 2023/11/28 11:41:36.201420 [info] home: language is set to en
Nov 28 11:41:36 adguardhome AdGuardHome[262]: 2023/11/28 11:41:36.201905 [info] home: theme is set to light
Nov 28 12:24:47 adguardhome AdGuardHome[262]: 2023/11/28 12:24:47.649769 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 28 12:24:47 adguardhome AdGuardHome[262]: 2023/11/28 12:24:47.652892 [info] filtering: updated filter 1563800924: 1139261 bytes, 58087 rules
Nov 28 12:24:53 adguardhome AdGuardHome[262]: 2023/11/28 12:24:53.537439 [info] filtering: updated filter 1563800924; rule count: 58087 (was 58079)
Nov 28 14:25:01 adguardhome AdGuardHome[262]: 2023/11/28 14:25:01.735178 [error] filtering: updating filter from url "https://raw.githubusercontent.com/StevenBlack/hosts/master/alternates/fakenews-gambling/hosts": reading from url: got status code 503, want 200
Nov 28 14:25:09 adguardhome AdGuardHome[262]: 2023/11/28 14:25:09.019616 [error] filtering: updating filter from url "https://raw.githubusercontent.com/anudeepND/blacklist/master/adservers.txt": reading from url: got status code 503, want 200
Nov 28 14:25:09 adguardhome AdGuardHome[262]: 2023/11/28 14:25:09.113447 [error] filtering: updating filter from url "https://raw.githubusercontent.com/bigdargon/hostsVN/master/hosts": reading from url: got status code 503, want 200
Nov 28 14:25:09 adguardhome AdGuardHome[262]: 2023/11/28 14:25:09.195275 [error] filtering: updating filter from url "https://raw.githubusercontent.com/FadeMind/hosts.extras/master/add.Spam/hosts": reading from url: got status code 503, want 200
Nov 28 16:37:31 adguardhome AdGuardHome[262]: 2023/11/28 16:37:31.146794 [error] handling tcp: reading msg: reading len: read tcp 10.0.99.14:853->10.0.10.28:46944: read: connection reset by peer
Nov 28 16:37:45 adguardhome AdGuardHome[262]: 2023/11/28 16:37:45.837841 [error] handling tcp: reading msg: reading len: read tcp 10.0.99.14:853->10.0.10.28:47002: read: connection reset by peer
Nov 28 18:26:45 adguardhome AdGuardHome[262]: 2023/11/28 18:26:45.676286 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 28 18:26:45 adguardhome AdGuardHome[262]: 2023/11/28 18:26:45.678824 [info] filtering: updated filter 1563800924: 1139481 bytes, 58097 rules
Nov 28 18:26:52 adguardhome AdGuardHome[262]: 2023/11/28 18:26:52.378117 [info] filtering: updated filter 1563800924; rule count: 58097 (was 58087)
Nov 28 19:21:03 adguardhome AdGuardHome[262]: 2023/11/28 19:21:03.025997 [info] home: language is set to en
Nov 28 19:21:03 adguardhome AdGuardHome[262]: 2023/11/28 19:21:03.026354 [info] home: theme is set to light
Nov 28 20:12:13 adguardhome AdGuardHome[262]: 2023/11/28 20:12:13.565659 [info] home: language is set to en
Nov 28 20:12:13 adguardhome AdGuardHome[262]: 2023/11/28 20:12:13.566136 [info] home: theme is set to light
Nov 28 22:13:23 adguardhome AdGuardHome[262]: 2023/11/28 22:13:23.337976 [info] home: language is set to en
Nov 28 22:13:23 adguardhome AdGuardHome[262]: 2023/11/28 22:13:23.338326 [info] home: theme is set to light
Nov 29 00:27:34 adguardhome AdGuardHome[262]: 2023/11/29 00:27:34.788243 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 29 00:27:34 adguardhome AdGuardHome[262]: 2023/11/29 00:27:34.794252 [info] filtering: updated filter 1563800924: 1139839 bytes, 58117 rules
Nov 29 00:27:40 adguardhome AdGuardHome[262]: 2023/11/29 00:27:40.507107 [info] filtering: updated filter 1563800924; rule count: 58117 (was 58097)
Nov 29 06:28:23 adguardhome AdGuardHome[262]: 2023/11/29 06:28:23.655883 [info] filtering: saving contents of filter 1563800924 into "/root/data/filters/1563800924.txt"
Nov 29 06:28:23 adguardhome AdGuardHome[262]: 2023/11/29 06:28:23.659049 [info] filtering: updated filter 1563800924: 1139815 bytes, 58116 rules
Nov 29 06:28:29 adguardhome AdGuardHome[262]: 2023/11/29 06:28:29.781234 [info] filtering: updated filter 1563800924; rule count: 58116 (was 58117)
Nov 29 06:29:13 adguardhome AdGuardHome[262]: 2023/11/29 06:29:12.705927 [error] dnsproxy: upstream 10.0.99.1:5353 failed to exchange ;33.10.0.10.in-addr.arpa.        IN         PTR in 4.530390853s: dialing 10.0.99.1:5353 over udp: all dialers failed: dial udp 10.0.99.1:5353: i/o timeout
Nov 29 06:32:43 adguardhome AdGuardHome[262]: 2023/11/29 06:32:41.984270 [error] dnsproxy: upstream 10.0.99.1:5353 failed to exchange ;1.99.0.10.in-addr.arpa.        IN         PTR in 8.153616002s: dialing 10.0.99.1:5353 over udp: all dialers failed: dial udp 10.0.99.1:5353: i/o timeout
Nov 29 06:38:05 adguardhome AdGuardHome[262]: 2023/11/29 06:38:05.505737 [info] home: language is set to en
Nov 29 06:38:05 adguardhome AdGuardHome[262]: 2023/11/29 06:38:05.506027 [info] home: theme is set to light
```

**Comment by ishanjain28** [comment]
I have enabled verbose logs and I'll report back in a day or two if memory consumption keeps going up and it starts timing out.

**Comment by schzhn** [comment]
Duplicate of https://github.com/AdguardTeam/AdGuardHome/issues/6438
@timkgh, @ishanjain28, fixed in the the [edge release](https://github.com/AdguardTeam/AdGuardHome/blob/master/README.md#test-unstable-versions).

**Comment by timkgh** [comment]
Confirm fixed in `v0.108.0-a.778+0e4e243e`
